diff --git a/src/transcript.c b/src/transcript.c
index a603a34..20af212 100644
--- a/src/transcript.c
+++ b/src/transcript.c
@@ -449,8 +449,6 @@ transcript_error_t transcript_init(void) {
       RELEASE_LOCK();
       return TRANSCRIPT_INIT_DLFCN;
     }
-/* Disabled because of security risks! */
-#if TRANSCRIPT_DEBUG
     if ((transcript_path = getenv("TRANSCRIPT_PATH")) != NULL) {
       if ((transcript_path = _transcript_strdup(transcript_path)) != NULL) {
         for (search_path_element = ts_strtok(transcript_path, path_sep, &state);
@@ -458,7 +456,6 @@ transcript_error_t transcript_init(void) {
           add_search_dir(search_path_element);
       }
     }
-#endif
 #if 0
 		if ((transcript_path = getenv("HOME")) != NULL) {
 			if ((search_path_element = malloc(strlen(transcript_path) + 1 + 11 + 1)) != NULL) {
